dialogue-core/src/main/java/com/palantir/dialogue/core/QueueAttachments.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.core;
+
+import com.palantir.dialogue.Channel;
+import com.palantir.dialogue.RequestAttachmentKey;
+
+interface QueueAttachments {
```

Comment:
nit: I tend to prefer concrete classes to make it clear this isn't meant to be subclassed

Reply:
Done